src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+export const Person = ({ person }) => {
+  if (person.sex === 'f') {
```

Comment:
think you don't need this if else check, components looks the same so you can only check what sex person has and they pass it to paragraph with partenr info: e.g:

Add this statement before return ()
```
 const spouse = sex === 'f'
    ? 'husband'
    : 'wife';
```

And your partner info can look like this:

```
  {isMarried ? (
        <p className="Person__partner">
          {`${partnerName} is my ${spouse}`}
        </p>
      ) : (
        <p className="Person__partner">I am not married</p>
      )}
```